spec/index.bs

```diff
@@ -1323,11 +1323,15 @@ configuration is suitable for the application.
 
 <script type=idl>
 dictionary GPURequestAdapterOptions {
+    GPUAllowSoftwareOption allowSoftware = "last-resort";
     GPUPowerPreference powerPreference;
 };
-</script>
 
-<script type=idl>
+enum GPUAllowSoftwareOption {
```

Comment:
Why can't this just be a boolean? It's up to the UA to figure out when this should be preferred, anyway.

Reply:
For extensibility, see [1]

Reply:
I think, if you want to have rich controls like this, then the "allow" part becomes confusing. I.e. `allowSoftware = "always"` doesn't read to me like a thing that would *prefer* software over hardware adapters.

Reply:
with plural requestAdapters, I envisioned "always" would still list software adapters last, effectively keeping the same preference order but not hiding software adapters.